core/sql/optimizer/RelGrby.h

```diff
@@ -152,6 +154,9 @@ class GroupByAgg : public RelExpr
   inline void setGroupExpr(ValueIdSet &expr) { groupExpr_ = expr;}
   inline void addGroupExpr(ValueIdSet &expr) { groupExpr_ += expr;}
 
+  inline ValueIdList & extraOrderExpr() { return extraOrderExpr_; }
```

Comment:
No problem, only if it would have been easy... A better solution, if possible, could be a setExtraOrderExpr(const ValueIdList &newExtraOrder) method. But again, just wanted to share a thought.